mail/mu/Portfile

```diff
@@ -4,39 +4,49 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           elisp 1.0
 
-github.setup        djcb mu 1.0 v
```

Comment:
Please leave the setup here on top.

Reply:
I am using the suggestion of michaelld.
I also prefer in this way where we follow convention over configuration.

Reply:
we have already discussed about this in another PR and the end was: "maintainer preference". Anyway, for me, as I said, it is good to set `github.setup` after `PortGroup` to find immediately name and version. @michaelld argued that his prefer to have everything in the same place, version, checksums and revision, to facilitate the bump version process.